database_setup/README.md

```diff
@@ -0,0 +1,19 @@
+## Development Database Setup
+
+To create a test development DB,
```

Comment:
Had tried to move the database setup info into the main README, but it's fine if it stays here... just have a link to it on the main README under the same title.